SU-SWS/ace-casbsgryphon
# CAS-202: Spacing for full title

# READY FOR REVIEW

# Summary
- The Board of Directors full title was bleeding into the bio. Fixed with some padding. 

# Review By (Date)
- Now

# Urgency
- Now?

# Steps to Test

1. Go to the BoD page
2. Notice how much easier it is to parse that they have titles!

# Affected Projects or Products
- Launch

# Associated Issues and/or People
- CAS-202

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
